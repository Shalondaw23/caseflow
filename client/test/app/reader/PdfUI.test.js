import React from 'react';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { axe } from 'jest-axe';

import PdfUI from '../../../app/reader/PdfUI';
import ReduxBase from '../../../../client/app/components/ReduxBase';

describe('PdfUI', () => {
  const defaultProps = {
    doc: {
      filename: 'My PDF',
      id: 3,
      type: 'Form 8',
      receivedAt: '1/2/2017',
      content_url: '',
    },
    file: 'test.pdf',
    filteredDocIds: [1, 5],
    id: 'pdf',
    pdfWorker: 'noworker',
    documentPathBase: '/reader/appeal/reader_id1',
    showClaimsFolderNavigation: false,
    featureToggles: { search: true }
  };

  const mockState = {
    pdf: {
      documents: { 1: {}, 5: {}, 18: {} },
      documentErrors: [],
      pdfDocuments: {},
    },
    pdfViewer: {
      hidePdfSidebar: false
    },
    documentList: {
      filteredDocIds: [1, 5],
      searchCategoryHighlights: {},
    },
    annotationLayer: {
      isPlacingAnnotation: false
    },
    searchActionReducer: {
      extractedText: ''
    }
  };

  // Setup usually combines passed props and default props
  // This one implements an anti-pattern and mocks a reducer, I'm doing this for two reasons:
  // 1) It's still (slightly) better than the existing karma tests since we're trying to deprecate Karma.
  // 2) This grossness can likely be removed/fixed after the reader refactor changes/rewrites this component.
  const setup = (props) => {
    const rootReducer = jest.fn();

    rootReducer.mockReturnValue(mockState);

    return render(
      <ReduxBase reducer={rootReducer}>
        <PdfUI {...defaultProps} {...props} />;
      </ReduxBase>
    );
  };

  describe('render', () => {
    it('renders correctly', () => {
      const { container } = setup();

      expect(container).toMatchSnapshot();
    });

    // This fails, skipping because there is a major reader refactor
    // in progress that should fix the error that's being reported.
    it.skip('passes a11y testing', async () => {
      const { container } = setup();

      const results = await axe(container);

      expect(results).toHaveNoViolations();
    });

    it('correctly displays html elements', () => {

    });
  });

  describe('actions', () => {
    it('calls action on click of XXX', () => {});
  });
});
