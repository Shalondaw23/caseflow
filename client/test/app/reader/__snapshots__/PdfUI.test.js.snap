// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PdfUI render renders correctly 1`] = `
<div>
  <div
    class="cf-pdf-container"
    data-css-msnuvu=""
  >
    <div
      class="cf-pdf-header cf-pdf-toolbar"
    >
      <span
        data-css-1rxr0aa=""
        data-css-k5n44n=""
      />
      <span
        data-css-1ha0cgc=""
        data-css-k5n44n=""
      >
        <span
          class="category-icons-and-doc-type"
        >
          <span
            class="cf-pdf-doc-category-icons"
          />
          <span
            class="cf-pdf-doc-type-button-container"
          >
            <a
              aria-label="open document in new tab"
              class="link-matte link-overflow"
              data-css-1iv2spu=""
              href="/reader/appeal/reader/appeal/reader_id1/3"
              id="newTab"
              target="_blank"
              type="button"
            >
              <h1
                class="cf-pdf-vertically-center cf-non-stylized-header"
              >
                <span
                  title="Open in new tab"
                >
                  Form 8
                </span>
                <span
                  class="cf-pdf-external-link-icon"
                >
                  <svg
                    height="16px"
                    version="1.1"
                    viewBox="0 0 18 16"
                    width="18px"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      fill="none"
                      fill-rule="evenodd"
                      stroke="none"
                      stroke-width="1"
                    >
                      <g
                        fill="#fff"
                        fill-rule="nonzero"
                      >
                        <g
                          transform="translate(1.000000, 1.000000)"
                        >
                          <path
                            d="M12.2767813,8.17242825 L11.7057188,8.17242825 C11.6223125,8.17242825 11.553875,8.19969476 11.500625,8.25448292 C11.4469375,8.30895216 11.4202188,8.37876082 11.4202188,8.46378132 L11.4202188,11.377344 C11.4202188,11.7779863 11.2805313,12.1208747 11.001,12.4062005 C10.7214688,12.6914624 10.3853438,12.8339818 9.99275,12.8339818 L2.5695,12.8339818 C2.1769375,12.8339818 1.84084375,12.6914624 1.56125,12.4062005 C1.28171875,12.1209066 1.14196875,11.7780182 1.14196875,11.377344 L1.14196875,3.80200456 C1.14196875,3.40139408 1.2816875,3.05853759 1.56125,2.77324374 C1.84084375,2.487918 2.1769375,2.34533485 2.5695,2.34533485 L8.8505625,2.34533485 C8.93396875,2.34533485 9.00240625,2.31794077 9.055875,2.26337585 C9.10934375,2.20874715 9.1360625,2.13900228 9.1360625,2.05394989 L9.1360625,1.47114806 C9.1360625,1.38625513 9.10934375,1.31638269 9.055875,1.26184966 C9.00240625,1.20722096 8.93396875,1.17995444 8.8505625,1.17995444 L2.5695,1.17995444 C1.86175,1.17995444 1.2565,1.43641913 0.7538125,1.94931663 C0.2513125,2.46218223 0,3.07984055 0,3.80210023 L0,11.3774715 C0,12.0996993 0.2513125,12.7174533 0.75384375,13.2301595 C1.25653125,13.7430888 1.86178125,13.9996811 2.56953125,13.9996811 L9.99275,13.9996811 C10.7004688,13.9996811 11.3057813,13.7430888 11.8084063,13.2301595 C12.3110625,12.7174533 12.562375,12.0997312 12.562375,11.3774715 L12.562375,8.46400456 C12.562375,8.37888838 12.535625,8.30904784 12.4819375,8.25448292 C12.4284063,8.19969476 12.3599688,8.17242825 12.2767813,8.17242825 Z"
                            id="external-link-outline-box"
                          />
                          <path
                            d="M15.8009687,0.172974943 C15.6879375,0.0576264237 15.5539375,0 15.3994062,0 L10.8313437,0 C10.6767187,0 10.5428125,0.0576264237 10.429875,0.172974943 C10.3168438,0.288259681 10.26025,0.42484738 10.26025,0.58267426 C10.26025,0.740501139 10.3168438,0.877120729 10.429875,0.992469248 L12.0000938,2.59491116 L6.183,8.53138952 C6.1234375,8.59207745 6.09375,8.661918 6.09375,8.74068793 C6.09375,8.81974487 6.12353125,8.88955353 6.183,8.95024146 L7.200125,9.98809112 C7.25959375,10.048779 7.32803125,10.0790433 7.40534375,10.0790433 C7.48265625,10.0790433 7.551125,10.0488109 7.6105625,9.98809112 L13.4276875,4.05170843 L14.9980313,5.65415034 C15.1109687,5.76949886 15.2448438,5.82718907 15.3995,5.82718907 C15.5541562,5.82718907 15.6880312,5.76949886 15.8010937,5.65415034 C15.914125,5.53883371 15.9705937,5.4022779 15.9705937,5.24445103 L15.9705937,0.58267426 C15.9705625,0.424751708 15.9139063,0.288259681 15.8009687,0.172974943 Z"
                            id="external-link-outline-arrow"
                          />
                        </g>
                      </g>
                    </g>
                  </svg>
                </span>
              </h1>
            </a>
          </span>
        </span>
      </span>
      <span
        data-css-k5n44n=""
        data-css-p29gji=""
      >
        <span
          class="cf-pdf-button-text"
        >
          Zoom:
        </span>
        <span>
          <button
            aria-label="zoom out"
            class="cf-pdf-button cf-pdf-spaced-buttons usa-button"
            id="button-zoomOut"
            type="button"
          >
            <i
              aria-hidden="true"
              class="fa fa-minus"
            />
          </button>
        </span>
        <span>
          <button
            aria-label="zoom in"
            class="cf-pdf-button cf-pdf-spaced-buttons usa-button"
            id="button-zoomIn"
            type="button"
          >
            <i
              aria-hidden="true"
              class="fa fa-plus"
            />
          </button>
        </span>
        <span>
          <button
            aria-label="fit to screen"
            class="cf-pdf-button cf-pdf-spaced-buttons usa-button"
            id="button-fit"
            type="button"
          >
            <svg
              height="19px"
              version="1.1"
              viewBox="0 0 18 19"
              width="18px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="1"
              >
                <g
                  fill="#fff"
                >
                  <g
                    transform="translate(10.000000, 1.000000)"
                  >
                    <path
                      d="M6.96306644,4.98252278 C6.8903554,5.0100459 6.82008669,5.02382192 6.75299597,5.02382192 C6.60192414,5.02382192 6.47586421,4.97140935 6.37496329,4.86684467 L5.16527048,3.67701267 L1.77284292,7.01384128 C1.66629226,7.1184349 1.54037945,7.17084747 1.39472196,7.17084747 C1.24900562,7.17084747 1.12297511,7.1184349 1.01657158,7.01384128 L0.15966415,6.17089891 C0.0532311919,6.06642106 0,5.94223422 0,5.79909088 C0,5.65600542 0.0532311919,5.53199224 0.15966415,5.42739862 L3.55235654,2.09051213 L2.34257545,0.900680132 C2.16890464,0.740924522 2.12973896,0.548233789 2.22490185,0.322289578 C2.32006474,0.107516683 2.48537864,0 2.72060813,0 L6.7530254,0 C6.89862404,0 7.02468397,0.0523836329 7.13126406,0.157093016 C7.23757931,0.261686634 7.29078108,0.385670879 7.29078108,0.528901045 L7.29078108,4.49503664 C7.29075166,4.72639286 7.18137612,4.88881107 6.96306644,4.98252278 Z"
                      id="Path"
                    />
                  </g>
                  <g
                    transform="translate(4.645391, 13.585366) scale(-1, -1) translate(-4.645391, -13.585366) translate(1.000000, 10.000000)"
                  >
                    <path
                      d="M6.96306644,4.98252278 C6.8903554,5.0100459 6.82008669,5.02382192 6.75299597,5.02382192 C6.60192414,5.02382192 6.47586421,4.97140935 6.37496329,4.86684467 L5.16527048,3.67701267 L1.77284292,7.01384128 C1.66629226,7.1184349 1.54037945,7.17084747 1.39472196,7.17084747 C1.24900562,7.17084747 1.12297511,7.1184349 1.01657158,7.01384128 L0.15966415,6.17089891 C0.0532311919,6.06642106 0,5.94223422 0,5.79909088 C0,5.65600542 0.0532311919,5.53199224 0.15966415,5.42739862 L3.55235654,2.09051213 L2.34257545,0.900680132 C2.16890464,0.740924522 2.12973896,0.548233789 2.22490185,0.322289578 C2.32006474,0.107516683 2.48537864,0 2.72060813,0 L6.7530254,0 C6.89862404,0 7.02468397,0.0523836329 7.13126406,0.157093016 C7.23757931,0.261686634 7.29078108,0.385670879 7.29078108,0.528901045 L7.29078108,4.49503664 C7.29075166,4.72639286 7.18137612,4.88881107 6.96306644,4.98252278 Z"
                      id="Path"
                    />
                  </g>
                </g>
              </g>
            </svg>
          </button>
        </span>
        <span>
          <button
            aria-label="rotate document"
            class="cf-pdf-button cf-pdf-spaced-buttons usa-button"
            id="button-rotation"
            type="button"
          >
            <svg
              height="19px"
              viewBox="0 0 19 19"
              width="19px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="1"
              >
                <g
                  fill="#fff"
                  fill-rule="nonzero"
                  transform="translate(9.313708, 9.313708) rotate(45.000000) translate(-9.313708, -9.313708) translate(1.813708, 0.813708)"
                >
                  <path
                    d="M7.57673077,16.3067308 C3.50307692,16.3067308 0.189230769,12.9894231 0.189230769,8.91692308 C0.189230769,4.84384615 3.50307692,1.52826923 7.57673077,1.52826923 C7.84730769,1.52826923 8.12076923,1.54384615 8.3925,1.57442308 L8.3925,0.380192308 C8.3925,0.226730769 8.48711538,0.0871153846 8.62673077,0.0276923077 C8.77211538,-0.0288461538 8.93307692,0.00288461539 9.04384615,0.113076923 L11.6613462,2.73634615 C11.8101923,2.88519231 11.8101923,3.12288462 11.6613462,3.27403846 L9.045,5.90307692 C8.93365385,6.01153846 8.77211538,6.045 8.62673077,5.98557692 C8.48711538,5.92557692 8.3925,5.78711538 8.3925,5.63423077 L8.3925,4.30961538 C8.12192308,4.26 7.84961538,4.2375 7.57673077,4.2375 C4.99615385,4.2375 2.89615385,6.33692308 2.89615385,8.91692308 C2.89615385,11.4963462 4.99673077,13.5975 7.57673077,13.5975 C10.1567308,13.5975 12.2561538,11.4957692 12.2561538,8.91692308 C12.2561538,8.7075 12.4263462,8.53615385 12.6380769,8.53615385 L14.5817308,8.53615385 C14.7923077,8.53615385 14.9636538,8.7075 14.9636538,8.91692308 C14.9636538,12.9894231 11.6475,16.3067308 7.57673077,16.3067308 Z"
                    id="Shape"
                  />
                </g>
              </g>
            </svg>
          </button>
        </span>
        <span
          class="cf-pdf-spaced-buttons"
        >
          |
        </span>
        <span>
          <button
            aria-label="download pdf"
            class="cf-pdf-button cf-pdf-download-icon usa-button"
            id="button-download"
            type="button"
          >
            <svg
              height="19px"
              version="1.1"
              viewBox="0 0 14 19"
              width="14px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="1"
              >
                <g
                  fill="#fff"
                >
                  <path
                    d="M14,15.6826233 C14,15.1556346 13.5724432,14.7280778 13.0454545,14.7280778 L9.90909091,14.7280778 L6.09090909,14.7280778 L1.95454545,14.7280778 C1.42755682,14.7280778 1,15.1556346 1,15.6826233 L1,16.454385 C1,16.9813736 1.42755682,17.4089304 1.95454545,17.4089304 L6.09090909,17.4089304 L9.90909091,17.4089304 L13.0454545,17.4089304 C13.5724432,17.4089304 14,16.9813736 14,16.454385 L14,15.6826233 Z"
                    id="Expand"
                  />
                  <g
                    transform="translate(7.362992, 6.091070) rotate(134.000000) translate(-7.362992, -6.091070) translate(2.862992, 1.591070)"
                  >
                    <path
                      d="M7.97587683,5.70725389 C7.89258963,5.73878036 7.81210001,5.75456018 7.73525063,5.75456018 C7.5622047,5.75456018 7.41780876,5.69452395 7.30223134,5.57474986 L5.91658308,4.21185126 L2.03071117,8.0340371 C1.90866222,8.15384435 1.76443481,8.21388058 1.59759076,8.21388058 C1.4306793,8.21388058 1.28631706,8.15384435 1.16443664,8.0340371 L0.182888043,7.06848485 C0.0609739162,6.9488102 -3.27418093e-11,6.80655982 -3.27418093e-11,6.64259561 C-3.27418093e-11,6.47869771 0.0609739162,6.33664623 0.182888043,6.21683898 L4.06906331,2.39458684 L2.68331394,1.03168824 C2.4843819,0.848695439 2.43951939,0.627976942 2.54852417,0.369168095 C2.65752895,0.123155484 2.84688852,9.50350909e-14 3.11633323,9.50350909e-14 L7.73528434,9.50350909e-14 C7.90206098,9.50350909e-14 8.04645692,0.0600030758 8.16853957,0.179942926 C8.29031888,0.299750172 8.35125909,0.441768502 8.35125909,0.605832161 L8.35125909,5.14886062 C8.35122538,5.41386868 8.22594067,5.59991137 7.97587683,5.70725389 Z"
                      id="Path"
                    />
                  </g>
                </g>
              </g>
            </svg>
          </button>
        </span>
        <span>
          <button
            aria-label="search text"
            class="cf-pdf-button cf-pdf-search usa-search usa-search-small usa-button"
            id="submit-search"
            type="submit"
          >
            <svg
              height="24"
              viewBox="0 0 216 146"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M172.77 123.025L144.825 95.08c6.735-9.722 10.104-20.56 10.104-32.508 0-7.767-1.51-15.195-4.527-22.283-3.014-7.09-7.088-13.2-12.22-18.336s-11.243-9.207-18.33-12.22c-7.09-3.016-14.52-4.523-22.286-4.523-7.768 0-15.196 1.508-22.284 4.523-7.09 3.014-13.2 7.088-18.332 12.22-5.132 5.134-9.206 11.245-12.22 18.333-3.015 7.088-4.522 14.515-4.522 22.282 0 7.766 1.507 15.192 4.522 22.282 3.014 7.088 7.088 13.197 12.22 18.33 5.134 5.134 11.245 9.207 18.333 12.222 7.09 3.015 14.516 4.522 22.283 4.522 11.95 0 22.786-3.37 32.51-10.105l27.943 27.863c1.955 2.064 4.397 3.096 7.332 3.096 2.824 0 5.27-1.03 7.332-3.096 2.064-2.063 3.096-4.508 3.096-7.332 0-2.877-1.002-5.322-3.013-7.33zm-49.413-34.668C116.214 95.5 107.62 99.07 97.57 99.07c-10.048 0-18.643-3.57-25.786-10.713C64.64 81.214 61.07 72.62 61.07 62.57c0-10.047 3.572-18.643 10.714-25.785C78.926 29.642 87.522 26.07 97.57 26.07c10.048 0 18.643 3.573 25.787 10.715 7.143 7.142 10.715 15.738 10.715 25.786 0 10.05-3.573 18.647-10.715 25.79z"
                fill="#fff"
              />
            </svg>
          </button>
        </span>
      </span>
    </div>
    <div>
      <div
        class="cf-search-bar"
      >
        <span
          class="usa-search usa-search-small cf-search-ahead cf-has-internal-text"
          role="search"
        >
          <label
            class="usa-sr-only"
            for="search-ahead"
          >
            Search
          </label>
          <input
            class="cf-search-input-with-close cf-search-with-internal-text"
            id="search-ahead"
            name="search"
            placeholder="Type to search..."
            type="search"
            value=""
          />
          <div>
            <label
              class="usa-sr-only"
              for="search-internal-text"
            >
              Search Result Count
            </label>
            <input
              class="cf-search-internal-text"
              id="search-internal-text"
              readonly=""
              type="text"
              value=""
            />
          </div>
        </span>
        <span>
          <button
            class="cf-increment-search-match cf-prev-match usa-button"
            type="button"
          >
            <div
              style="transform: translateY(5px) translateX(-0.5rem);"
            >
              <svg
                class="fa-chevron-left"
                height="17px"
                version="1.1"
                viewBox="0 0 11 17"
                width="11px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                  stroke="none"
                  stroke-width="1"
                >
                  <g
                    fill="#fff"
                    fill-rule="nonzero"
                    id="chevron-left-white"
                  >
                    <path
                      d="M10.148056,7.63345458 L3.83667957,1.36581253 C3.60425101,1.12190379 3.31401199,1 2.9653861,1 C2.6167602,1 2.32652118,1.12203868 2.09409262,1.36581253 L1.35839094,2.08799554 C1.11955406,2.32549717 1.00013563,2.61418827 1.00013563,2.95437235 C1.00013563,3.28811559 1.11952016,3.5802463 1.35839094,3.83049472 L6.06300954,8.50003372 L1.3581875,13.1791834 C1.11941844,13.416685 1,13.7053761 1,14.0455939 C1,14.3793034 1.11938453,14.6715353 1.3581875,14.9216826 L2.09395699,15.6437644 C2.33275996,15.881266 2.62330414,16 2.96525047,16 C3.30733242,16 3.59780879,15.881266 3.83654395,15.6437644 L10.148056,9.37612237 C10.3869607,9.12580651 10.5064808,8.83374324 10.5064808,8.5 C10.5065147,8.15981592 10.3869607,7.87095621 10.148056,7.63345458 Z"
                      id="Shape"
                      transform="translate(5.753240, 8.500000) scale(-1, 1) translate(-5.753240, -8.500000) "
                    />
                  </g>
                </g>
              </svg>
              <span
                class="usa-sr-only"
              >
                Previous Match
              </span>
            </div>
          </button>
        </span>
        <span>
          <button
            class="cf-increment-search-match cf-next-match usa-button"
            type="button"
          >
            <div
              style="transform: translateY(5px) translateX(-0.5rem);"
            >
              <svg
                class="fa-chevron-left"
                height="17px"
                version="1.1"
                viewBox="0 0 11 17"
                width="11px"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                  stroke="none"
                  stroke-width="1"
                >
                  <g
                    fill="#fff"
                    fill-rule="nonzero"
                    id="chevron-left-white"
                  >
                    <path
                      d="M10.148056,7.63345458 L3.83667957,1.36581253 C3.60425101,1.12190379 3.31401199,1 2.9653861,1 C2.6167602,1 2.32652118,1.12203868 2.09409262,1.36581253 L1.35839094,2.08799554 C1.11955406,2.32549717 1.00013563,2.61418827 1.00013563,2.95437235 C1.00013563,3.28811559 1.11952016,3.5802463 1.35839094,3.83049472 L6.06300954,8.50003372 L1.3581875,13.1791834 C1.11941844,13.416685 1,13.7053761 1,14.0455939 C1,14.3793034 1.11938453,14.6715353 1.3581875,14.9216826 L2.09395699,15.6437644 C2.33275996,15.881266 2.62330414,16 2.96525047,16 C3.30733242,16 3.59780879,15.881266 3.83654395,15.6437644 L10.148056,9.37612237 C10.3869607,9.12580651 10.5064808,8.83374324 10.5064808,8.5 C10.5065147,8.15981592 10.3869607,7.87095621 10.148056,7.63345458 Z"
                      id="Shape"
                      transform="translate(5.753240, 8.500000) scale(1, 1) translate(-5.753240, -8.500000) "
                    />
                  </g>
                </g>
              </svg>
              <span
                class="usa-sr-only"
              >
                Next Match
              </span>
            </div>
          </button>
        </span>
      </div>
      <div
        class="cf-pdf-scroll-view"
      >
        <div
          id=""
          style="position: relative; width: 100%; height: 100%;"
        />
      </div>
    </div>
    <div
      class="cf-pdf-footer cf-pdf-toolbar"
      data-css-1q47s6r=""
    >
      <div
        class="cf-pdf-footer-buttons-left"
      />
      <div
        class="cf-pdf-buttons-center"
      >
        <span>
          <span
            class="page-progress-indicator"
          >
            <em>
              Loading document...
            </em>
          </span>
          |
        </span>
        <span
          class="doc-list-progress-indicator"
        >
          <svg
            class="filter-icon"
            height="14px"
            viewBox="0 0 12 14"
            width="12px"
          >
            <g
              fill="none"
              fill-rule="evenodd"
              stroke="none"
              stroke-width="1"
            >
              <g
                fill="#fff"
                fill-rule="nonzero"
              >
                <g
                  id="filter-filled-tool-symbol"
                  transform="translate(6.000000, 7.000000) scale(-1, 1) translate(-6.000000, -7.000000) translate(0.000000, 1.000000)"
                >
                  <path
                    d="M4.55870539,5.67037037 C4.68567635,5.80740741 4.75538589,5.98641975 4.75538589,6.17160494 L4.75538589,11.6283951 C4.75538589,11.9568025 5.15497095,12.1234691 5.39148548,11.8925926 L6.92634025,10.1481481 C7.13173444,9.9037037 7.24501245,9.7827037 7.24501245,9.54074074 L7.24501245,6.17283951 C7.24501245,5.98765432 7.3159668,5.80864198 7.44169295,5.67159259 L11.8458299,0.932098765 C12.1757054,0.57654321 11.9217759,0 11.4337967,0 L0.566576763,0 C0.0786099585,0 -0.176576763,0.575308642 0.154543568,0.932098765 L4.55870539,5.67037037 Z"
                  />
                </g>
              </g>
            </g>
          </svg>
          Document 
          0
           of 
          2
        </span>
      </div>
      <div
        class="cf-pdf-footer-buttons-right"
      />
    </div>
  </div>
  ;
</div>
`;
